content/automate/ManageIQ/Infrastructure/VM/Transform/StateMachines/VmImport.class/__methods__/submit_vm_import.rb

```diff
@@ -0,0 +1,40 @@
+#
+# Description: This method initiates VM import to given infra provider
+#
+
+module ManageIQ
+  module Automate
+    module Infrastructure
+      module VM
+        module Transform
+          module StateMachines
+            class SubmitVmImport
+              def initialize(handle = $evm)
+                @handle = handle
+              end
+
+              def main
+                provider = @handle.vmdb(:ext_management_system, @handle.root['provider_id'])
+                @handle.log(:info, 'Submitting Import')
+                new_ems_ref = provider.submit_import_vm(
```

Comment:
@tinaafitz Defined in https://github.com/ManageIQ/manageiq-providers-ovirt/pull/21 and exposed in https://github.com/ManageIQ/manageiq-automation_engine/pull/4